alt-tab-macos/logic/TrackedWindow.swift

```diff
@@ -42,4 +42,17 @@ class TrackedWindow {
         }
         axWindow?.focus(id)
     }
+
+    func close() {
+        if axWindow == nil {
+            axWindow = id.AXUIElementOfOtherSpaceWindow(ownerPid)
+        }
+        axWindow?.close()
+    }
+
+    func quitApp() {
+        if app != nil {
+            app!.terminate()
+        }
```

Comment:
I would suggest a simpler `app?.terminate()` here

Reply:
Good catch, I'll change that